orm/model_table_struct.go

```diff
@@ -362,15 +363,29 @@ func (m *structTableModel) join(
 func (m *structTableModel) setSoftDeleteField() {
 	field := m.table.SoftDeleteField
 	value := field.Value(m.strct)
+	kind := value.Kind()
 
+	unixNow := time.Now().UnixNano() / 1e6
```

Comment:
This should be `/ time.Millisecond`. Overall - why do you use milliseconds precision? Why not to use `time.Now().UnixNano()` as is?

Reply:
I always use milliseconds, but yours is a legit question, let me know if you want me to change that.

Reply:
Yes, please change it to use nanoseconds just like Go - that is what most Go users would expect (including me). Otherwise we just losing precision without saving any space...

Reply:
Done, waiting for review.